app/assets/stylesheets/_helper_classes.scss

```diff
@@ -54,3 +65,15 @@ label[for$=_unknown] {
 .text-error {
   color: red;
 }
+
+.pull-right-content {
+  float: right;
+}
+
+.complete-label {
```

Comment:
Sure happy to have a discussion about it. It may apply to other places as well. Shell we have it now or when in a less busy period?